motoman_driver/MotoPlus/Controller.h

```diff
@@ -58,6 +58,12 @@
 #define MAX_MOTION_CONNECTIONS	1
 #define MAX_STATE_CONNECTIONS	4
 
+#if (DX100)
+#define MAX_CONTROLLABLE_GROUPS	2

```

Comment:
Limiting the group-count to 2 for DX100 is not good as I explained in [using-the-motoman-sda20d-dual-arms-dx100-ros](https://discourse.ros.org/t/using-the-motoman-sda20d-dual-arms-dx100-ros-interface-indigo/4667/19).

Reply:
Interesting... I need to investigate.